### DISCUSSION TITLE
presto 0.217 giving org.apache.parquet.io.PrimitiveColumnIO cannot be cast to org.apache.parquet.io.GroupColumnIO

### DISCUSSION THREAD

**Discussion by kishoreduraisam** [prestodb/presto#15507] [Q&A]
I have a table pointing to s3 location. The s3 locations have parquet files. We have multiple large files partitioned by day in the s3 location.  I have a column in the table which is expected to have an array<String>. But one of the files in the table does not have any non-null records for this column. So when I try to query that column, I am getting this exception org.apache.parquet.io.PrimitiveColumnIO cannot be cast to org.apache.parquet.io.GroupColumnIO. Is there a solution to get around it? @findepi

**Comment by rschlussel** [comment]
@vkorukanti @zhenxiao Can you help with this question?

**Comment by zhenxiao** [comment]
hi @kishoreduraisam could you please show example code and data? seems like a parquet schema and table schema mismatch problem.

**Comment by kishoreduraisam** [comment]
@zhenxiao @rschlussel @vkorukanti  I was able to figure this out. It was the parquet schema and table schema mismatch. Once I sorted that out, I am good. Thanks for helping.
